backends/tfhe-cuda-backend/cuda/src/integer/integer.cuh

```diff
@@ -476,14 +474,18 @@ void scratch_cuda_full_propagation(
   // PBS
   int8_t *pbs_buffer;
   if (pbs_type == MULTI_BIT) {
-    uint32_t lwe_chunk_size = get_average_lwe_chunk_size(
-        lwe_dimension, pbs_level, glwe_dimension, num_radix_blocks);
-    // Only 64 bits is supported
-    scratch_cuda_multi_bit_pbs_64(stream, &pbs_buffer, lwe_dimension,
-                                  glwe_dimension, polynomial_size, pbs_level,
-                                  grouping_factor, num_radix_blocks,
-                                  cuda_get_max_shared_memory(stream->gpu_index),
-                                  allocate_gpu_memory, lwe_chunk_size);
+    if (sizeof(Torus) == sizeof(uint32_t)) {
```

Comment:
The same applies here. Since we only support 64 bits, we should abort for anything different than that.

Reply:
Just to be sure you didn't miss this one: are we not replacing this `if` by a `switch` as in `execute_scratch_pbs`?

Reply:
Totally forgot it, thanks :slightly_smiling_face: